docs/configuration-process.asciidoc

```diff
@@ -117,6 +117,25 @@ Defaults to `debug/vars`.
 Experimental Metrics endpoint for collecting application metrics.
 Enabled by default.
 
+[[register.ingest.pipeline]]
+[float]
+==== `register.ingest.pipeline`
+A pipeline is a definition of processors registered in Elasticsearch,
```

Comment:
Pipelines are already described in `configuring-ingest-node`,  and eg. here: https://www.elastic.co/guide/en/elasticsearch/reference/master/pipeline.html
maybe link to that instead of repeating?